src/stpipe/step.py

```diff
@@ -576,6 +576,15 @@ def call(cls, *args, **kwargs):
         if 'class' in config:
             del config['class']
 
+        if 'logcfg' in config:
+            try:
+                log.load_configuration(config['logcfg'])
+            except Exception as e:
+                raise ValueError(
+                    f"Error parsing logging config {config['logcfg']}:\n{e}"
+                )
+            del config['logcfg']
+
```

Comment:
The traceback produced here will be a bit confusing.

Slightly better exception handling would be to raise the 2nd except `from` the first.  Then it will be clear what caused what in the traceback.  And you don't have to try to include the handled exception inside the one your'e raising.

Also, not sure `ValueError` is appropriate.  Maybe `RuntimeError` would be better?

```suggestion
            except Exception as e:
                raise RuntimeError(
                    f"Error parsing logging config {config['logcfg']}"
                ) from e
            del config['logcfg']

```